.zenodo.json

```diff
@@ -187,6 +192,9 @@
       "name": "Klug, Julian",
       "orcid": "0000-0002-4849-9811"
     },
+    {
+      "name": "Vos de Wael, Reinder"
```

Comment:
@ReinderVosDeWael Would you like to add an affiliation or ORCID? This will be part of the Zenodo author listing on the next release.